pages/onboarding/congratulations.tsx

```diff
@@ -1,24 +1,33 @@
 import styled from 'styled-components';
 import { BaseM, TitleL } from 'components/core/Text/Text';
-import FullPageCenteredStep from 'flows/shared/components/FullPageCenteredStep/FullPageCenteredStep';
-import { usePreloadedQuery } from 'react-relay';
-import { useWizardState } from 'contexts/wizard/WizardDataProvider';
-import { organizeCollectionQuery } from 'flows/shared/steps/OrganizeCollection/OrganizeCollection';
+import FullPageCenteredStep from 'flows/../../src/components/Onboarding/FullPageCenteredStep/FullPageCenteredStep';
+import { useLazyLoadQuery } from 'react-relay';
 import { ButtonLink } from 'components/core/Button/Button';
 import { VStack } from 'components/core/Spacer/Stack';
+import { graphql } from 'relay-runtime';
+import { congratulationsQuery } from '../../__generated__/congratulationsQuery.graphql';
+
+export default function Congratulations() {
+  const query = useLazyLoadQuery<congratulationsQuery>(
+    graphql`
+      query congratulationsQuery {
+        viewer {
+          ... on Viewer {
+            __typename
+
+            user {
+              username
+            }
+          }
+        }
+      }
+    `,
+    {}
+  );
 
-function Congratulations() {
-  const { queryRef } = useWizardState();
-
-  if (!queryRef) {
-    throw new Error('Congratulations.tsx could not access queryRef');
-  }
-
-  const query = usePreloadedQuery(organizeCollectionQuery, queryRef);
-
-  if (query.viewer.__typename !== 'Viewer') {
+  if (query.viewer?.__typename !== 'Viewer') {
     throw new Error(
-      `OrganizeCollection expected Viewer to be type 'Viewer' but got: ${query.viewer.__typename}`
+      `OrganizeCollection expected Viewer to be type 'Viewer' but got: ${query.viewer?.__typename}`
```

Comment:
nit: error message should probably say `Congratulations expected Viewer ...`

Reply:
true